src/profile-logic/tracks.js

```diff
@@ -616,18 +622,24 @@ function _isContentThreadWithNoPaint(thread: Thread): boolean {
   return false;
 }
 
-// Any thread with less than 5% non-idle time will be hidden.
+// Any thread, except content thread with no RefreshDriverTick, with less than
+// 5% non-idle time will be hidden.
 const PERCENTAGE_ACTIVE_SAMPLES = 0.05;
 
+// Any content thread with no RefreshDriverTick with less than 20% non-idle
+// time will be hidden.
+const PERCENTAGE_ACTIVE_SAMPLES_NON_PAINT = 0.2;
```

Comment:
I'm not happy with naming here, this is the best that I could come up with :/ Do you have any suggestions?

Reply:
I think it's fine for me, especially since there is a good code comment.